package.json

```diff
@@ -133,12 +133,17 @@
       "name": "mafmt",
       "hash": "QmWLfU4tstw2aNcTykDm44xbSTCYJ9pUJwfhQCKGwckcHx",
       "version": "0.0.0"
+    },
+    {
+      "author": "whyrusleeping",
+      "name": "gorocheck",
+      "hash": "QmWBTBj2ceC8gk2pS5scr4WwuHGrzsTkUnPpv1KPhYET57",
+      "version": "0.0.0"
```

Comment:
would be great to have the "pkg repo" (individual pkg repo where you can report issues) listed in gx to get it from stored here. this could be a link, http for now and ipns name in the future.